src/Provider/StackExchange.php

```diff
@@ -76,7 +76,7 @@ public function getUserProfile()
     {
         $site = $this->config->get('site');
 
-        $response = $this->apiRequest('me?site=' . $site);
+        $response = $this->apiRequest('me?site=' . $site, 'GET', [ 'access_token' => $this->getStoredData('access_token') ]);
```

Comment:
it makes sense also to move `site` to the last argument

Reply:
Fixed. NB: I did test this worked.